test/metabase/test/util.clj

```diff
@@ -697,7 +697,8 @@
   (test-runner.parallel/assert-test-is-not-parallel "with-model-cleanup")
   (initialize/initialize-if-needed! :db)
   (let [model->old-max-id (into {} (for [model models]
-                                     [model (:max-id (db/select-one [model [:%max.id :max-id]]))]))]
+                                     [model (:max-id (db/select-one [model [(keyword (str "%max." (name (models/primary-key model))))
+                                                                            :max-id]]))]))]
```

Comment:
Didn't end up using this but I tweaked `with-model-cleanup` to work for other models that have sequential non-composite integer primary keys other than `:id`. So this should work for something like `CardEmitter` since Toucan treats `:emitter_id` as its primary key.